software/messaging/src/test/java/brooklyn/entity/messaging/rabbit/RabbitEc2LiveTest.java

```diff
@@ -81,7 +81,16 @@ private Channel getAmqpChannel(RabbitBroker rabbit) throws Exception {
         Channel channel = conn.createChannel();
         return channel;
     }
-    
+
+    @Override
+    public void test_CentOS_5_6() throws Exception {
+        // Not supported. The EPEL repository described here at [1] does not contain erlang, and the
```

Comment:
Could throw a [`SkipException`](http://testng.org/javadoc/org/testng/SkipException.html) to be explicit to the build.